tasks/overlays-pipeline.yaml

```diff
@@ -160,6 +160,8 @@ spec:
                             param["value"] = overlay_name
                         if param["name"] == "overlays_dir":
                             param["value"] = overlays_dir
+                        if param["name"] == "base_image":
+                            param["value"] = overlay_env["build"]["base-image"]
```

Comment:
`base_image` vs `base-image` ?

Reply:
Yes in the .aicoe.yaml file they are with `-`, while as env variables in tekton template we use `_`

Reply:
hmm :)

Reply:
> hmm :)

we need to restructure them in any case I guess and migrate Python code to https://github.com/thoth-station/pipeline-helpers

Reply:
There is a loop above it for keys under `build`
https://github.com/AICoE/aicoe-ci/blob/788233c6aca45dfc2e5d563411a44be22fc1bdd1/tasks/overlays-pipeline.yaml#L149 
this takes care of `base-image` and all the rest of keys under `build`

Reply:
Thanks @harshad16! the PR is no more required!